be/src/olap/push_handler.cpp

```diff
@@ -842,9 +833,9 @@ Status PushBrokerReader::init(const Schema* schema, const TBrokerScanRange& t_sc
     BaseScanner* scanner = nullptr;
     switch (t_scan_range.ranges[0].format_type) {
     case TFileFormatType::FORMAT_PARQUET:
-        scanner = new ParquetScanner(_runtime_state.get(), _runtime_profile, t_scan_range.params,
```

Comment:
You can remove ParquetScanner from Doris after this.

Reply:
ok